src/git/remotes/gerrit.ts

```diff
@@ -191,4 +191,8 @@ export class GerritRemote extends RemoteProvider {
 		if (branch) return `${this.encodeUrl(`${this.getUrlForBranch(branch)}/${fileName}`)}${line}`;
 		return `${this.encodeUrl(`${this.baseUrl}/+/HEAD/${fileName}`)}${line}`;
 	}
+
+	protected override getUrlForTag(): string | undefined {
+		return undefined;
```

Comment:
Is there no support for tags?

Reply:
There is no tag details view for gerrit, only tags list view